index.html

```diff
@@ -13,11 +14,19 @@
         <header class="nav-bar">
             <a class="logo" href="#">Snoplezik</a>
             <a class="hamburger" href="#"><div></div><div></div><div></div></a>
+            <ul class="nav-list">
+                <li>Portfolio</li>
+                <li>About</li>
+                <li>Contact</li>
+                <li><a href=""><img src="images/desk-nav-bar/Icon-Mail.png" alt="Mail"></a></li>
+            </ul>
         </header>
         <section class="headline">
-            <h1>Hey There.<br>I'm Santiago</h1>
-            <h2>I am a Software Developer</h2>
-            <p>I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+            <div class="head">
+              <h1>Hey There.<br>I'm Santiago</h1>
+              <h2>I am a Software Developer</h2>
+              <p>I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+            </div>
             <ul class="social">
                 <li><a href=""><img src="images/GitHub/Tertiary/Icons/Frame.png" alt="GitHub"></a></li>
                 <li><a href=""><img src="images/Twitter/Tertiary/Icons/Frame.png" alt="Twitter"></a></li>
```

Comment:
Your icons order doesn't match that of the template design. Please review the Figma template and make sure you fix it.

<img width="571" alt="Screen Shot 2022-07-07 at 22 14 33" src="https://user-images.githubusercontent.com/82426682/177909954-45b33e33-d03d-40d2-881e-6ce1c3abec3b.png">